packages/itwin/grouping-mapping-widget/src/widget/components/GroupsView.tsx

```diff
@@ -98,10 +118,10 @@ export const GroupsView = ({
       </div>
       {alert}
       <div className='gmw-groups-border' />
-      {!!numberOfVisualizedGroups &&
+      {!!hilitedGroupsProgress &&
         <div className="gmw-group-progress-bar">
           <ProgressLinear
-            value={25 + (numberOfVisualizedGroups / groups.length * 65)}
+            value={baseProgress + (hilitedGroupsProgress.currentHilitedGroups / hilitedGroupsProgress.totalNumberOfGroups * maxDynamicProgress)}
```

Comment:
The value ranges from 25 to 90. Is that correct? Could you add a comment with reasoning why this is not 0-100?

Reply:
Added comment